.github/workflows/e2e-test-dispatch-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+name: ARC-REUSABLE-WORKFLOW
```

Comment:
This project's been using this style for naming workflows:

https://github.com/actions/actions-runner-controller/blob/1701d34799bcda7498cd1a58fff8c8ab41f70cdb/.github/workflows/publish-arc.yaml#LL1C18-L1C18

Maybe we continue using it? The name will appear in here: https://github.com/actions/actions-runner-controller/actions

Reply:
I see different styles there as well. Here is how it will look like with these 2, which personally I prefer as they are since we do not have a pattern, it helps me see them easier albeit for now
![Screenshot 2023-02-14 at 14 42 41](https://user-images.githubusercontent.com/11218015/218755980-728838e5-cbb3-40f6-a711-c8ed3752817d.png)

Reply:
Yeah you're right, we should do better :/ I'll leave it up to you, I'm not sure adding a 3rd style is the best way to go though